components/CustomPopover/index.js

```diff
@@ -77,31 +76,32 @@ const CustomPopover = ({
   return (
     <>
       <HelpOutlineIcon
+        name='custom-popover'
         className={classes.icon}
         ref={anchorRef}
         aria-controls={open ? 'menu-list-grow' : undefined}
-        aria-haspopup="true"
+        aria-haspopup='true'
         onClick={handleOpenActions}
         // onMouseEnter={handleOpenActions}
         // onMouseLeave={handleCloseActions}
       />
-      <ClickAwayListener onClickAway={handleClosePopover}>
       <Popper
         placement='top'
         open={open}
         anchorEl={anchorRef.current}
         role={undefined}
         transition
       >
-        <Paper
-          className={clsx(classes.paper, contentClassName)}
-          // onMouseEnter={handleEnterPopover}
-          // onMouseLeave={handleClosePopover}
-        >
-          {children}
-        </Paper>
+        <ClickAwayListener onClickAway={handleClosePopover}>
+          <Paper
+            className={clsx(classes.paper, contentClassName)}
+            // onMouseEnter={handleEnterPopover}
```

Comment:
Lets make sure we add a description for why the code is being commented out, leave commented out code only if needs to be re-introduce, otherwise please remove it.